dojo/importers/reimporter/reimporter.py

```diff
@@ -300,15 +302,16 @@ def process_parsed_findings(self, test, parsed_findings, scan_type, user, active
             if create_finding_groups_for_all_findings or len(findings) > 1:
                 for finding in findings:
                     finding_helper.add_finding_to_auto_group(finding, group_by, **kwargs)
+
             if push_to_jira:
                 if findings[0].finding_group is not None:
                     jira_helper.push_to_jira(findings[0].finding_group)
                 else:
                     jira_helper.push_to_jira(findings[0])
 
-        if is_finding_groups_enabled() and push_to_jira:
-            for finding_group in set([finding.finding_group for finding in reactivated_items + unchanged_items if finding.finding_group is not None]):
-                jira_helper.push_to_jira(finding_group)
+            if is_finding_groups_enabled() and push_to_jira:
+                for finding_group in set([finding.finding_group for finding in reactivated_items + unchanged_items if finding.finding_group is not None]):
+                    jira_helper.push_to_jira(finding_group)
```

Comment:
I'm not sure you should modify this. If really sending data to JIRA break because some data are not persistented, this part should be extracted and done after `post_process_finding_save` method. but It could not be the case.

Reply:
Yeah, that change was bogus - a leftover from some desperate testing